### ISSUE TITLE
[BUG] Current balance doesn't change even after getting fills in paper_trade mode

### ISSUE THREAD

**Issue by RC-13** [CoinAlpha/hummingbot#2633]
**Describe the bug**
Current balance on paper_trade doesn't adjust accordingly after getting filled trades. The original balance remained as current even having more than 100 trades.

**Steps To Reproduce**
// A concise description of the **steps to reproduce the buggy behavior**:

Here are the steps to reproduce the issue (see attachments in the section below):
1. Enable paper trading mode
2. Create a pure market making strategy or cross exchange strategy
3. Start bot 
4. Once you get a filled trade, run `history` command and check current balance and notice that it doesn't change.
...

**Screenshots**
![im1](https://user-images.githubusercontent.com/57189990/99161339-54390780-272c-11eb-9d7e-a1bfa04416a0.png)
![image](https://user-images.githubusercontent.com/57189990/99161255-5e0e3b00-272b-11eb-8f34-7870aed3f7dc.png)

**Release version**
0.33.0

**Attachments**

*WARNING: Do NOT publish any exchange API keys or your wallet's private key. Whoever has access to them may steal your assets!*

**Optional: your discord username:**

**Comment by zappra** [comment]
See also #2511

**Comment by ebirbe** [comment]
> See also #2511

Thanks @zappra , I think its a different issue, because this one is not related to start/stop the bot.

Issue here is that `current` balance is never updated, and the `start` balance is what keeps changing all along the trades are being performed. In fact, if you see the first screenshot on the description, you will notice that even with 232 trades performed, the `current` balance still the same as I configured as initial balance, and the `start` balance is what changes over time.

I'd expect `start` balance to be the fixed value with what I used when started the bot, and the `current` balance would be what changes all the way while the bot is running.

**Comment by zappra** [comment]
Yes sorry! Perhaps I should have added a further comment, #2511 is a separate but related issue, with regard to the way the paper balances behave in response to trading activity. I'd like to see the balances behave "correctly" so that paper trading matches what you'd see in a live environment.

**Comment by Brunya** [comment]
+1
